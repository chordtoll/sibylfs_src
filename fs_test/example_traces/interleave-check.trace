@type trace

Pid 2 -> create

open_close "foo" [O_CREAT] 0o600
Tau
RV_none

Pid 1 -> unlink "foo"
Pid 2 -> unlink "foo"
Tau
Tau
Pid 1 <- RV_none
Pid 2 <- ENOENT
