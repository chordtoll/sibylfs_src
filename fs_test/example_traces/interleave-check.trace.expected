# processing file 'example_traces/interleave-check.trace' ...
        @type trace

     3: Pid 2 -> create (User_id 0) (Group_id 0)

     5: open_close "foo" [O_RDONLY;O_CREAT] 0o600
     6: Tau
     7: RV_none

     9: Pid 1 -> unlink "foo"
    10: Pid 2 -> unlink "foo"
    11: Tau
    12: Tau
    13: Pid 1 <- RV_none
# 
# Error:     13: Pid 1 <- RV_none
#    errors and values mixed in result
    14: Pid 2 <- ENOENT

# trace accepted
